Packages/MIES/MIES_BrowserSettingsPanel.ipf

```diff
@@ -996,33 +1003,56 @@ Window BrowserSettingsPanel() : Panel
 	PopupMenu popup_DB_lockedDevices,userdata(tabcontrol)=  "Settings"
 	PopupMenu popup_DB_lockedDevices,mode=1,popvalue="ITC18USB_Dev_0",value= #"DB_GetAllDevicesWithData()"
 	ListBox list_dashboard,pos={4.00,90.00},size={353.00,311.00},disable=1,proc=AD_ListBoxProc
-	ListBox list_dashboard,userdata(tabnum)=  "6",userdata(tabcontrol)=  "Settings"
+	ListBox list_dashboard,userdata(tabnum)=  "7",userdata(tabcontrol)=  "Settings"
 	ListBox list_dashboard,fSize=12,mode= 1,selRow= -1,widths={141,109,500}
 	ListBox list_dashboard,userColumnResize= 1
 	GroupBox group_enable_dashboard,pos={4.00,27.00},size={355.00,57.00},disable=1
-	GroupBox group_enable_dashboard,userdata(tabnum)=  "6"
+	GroupBox group_enable_dashboard,userdata(tabnum)=  "7"
 	GroupBox group_enable_dashboard,userdata(tabcontrol)=  "Settings"
 	CheckBox check_BrowserSettings_DB_Passed,pos={160.00,40.00},size={48.00,16.00},disable=1,proc=AD_CheckProc_PassedSweeps,title="Passed"
 	CheckBox check_BrowserSettings_DB_Passed,help={"Show passed sweeps on double click into ListBox "}
-	CheckBox check_BrowserSettings_DB_Passed,userdata(tabnum)=  "6"
+	CheckBox check_BrowserSettings_DB_Passed,userdata(tabnum)=  "7"
 	CheckBox check_BrowserSettings_DB_Passed,userdata(tabcontrol)=  "Settings"
 	CheckBox check_BrowserSettings_DB_Passed,value= 0
 	CheckBox check_BrowserSettings_DB_Failed,pos={160.00,60.00},size={43.00,16.00},disable=1,proc=AD_CheckProc_FailedSweeps,title="Failed"
 	CheckBox check_BrowserSettings_DB_Failed,help={"Show failed sweeps on double click into ListBox "}
-	CheckBox check_BrowserSettings_DB_Failed,userdata(tabnum)=  "6"
+	CheckBox check_BrowserSettings_DB_Failed,userdata(tabnum)=  "7"
 	CheckBox check_BrowserSettings_DB_Failed,userdata(tabcontrol)=  "Settings"
 	CheckBox check_BrowserSettings_DB_Failed,value= 0
+	GroupBox group_sweepFormula,pos={3.00,27.00},size={354.60,372.60},title="SweepFormula"
+	GroupBox group_sweepFormula,userdata(tabnum)=  "5",userdata(tabcontrol)=  "Settings"
+	SetVariable setvar_sweepFormula_formula,pos={18.60,70.20},size={300.00,13.80},proc=setvar_sweepFormula_update,title="formula"
+	SetVariable setvar_sweepFormula_formula,help={"Add a formula to display calculation results from sweeps."}
+	SetVariable setvar_sweepFormula_formula,limits={-inf,inf,0},live= 1
+	SetVariable setvar_sweepFormula_formula,userdata(tabnum)=  "5",userdata(tabcontrol)=  "Settings"
+	ValDisplay status_sweepFormula_parser,pos={319.80,70.80},size={16.80,13.80},bodyWidth=17
+	ValDisplay status_sweepFormula_parser,help={"Current parsing status of the entered formula."}
+	ValDisplay status_sweepFormula_parser,limits={-1,1,0},barmisc={0,0},mode= 1,highColor= (0,65535,0),lowColor= (0,0,0),zeroColor= (65535,0,0)
+	ValDisplay status_sweepFormula_parser,value= #"NaN"
+	ValDisplay status_sweepFormula_parser,userdata(tabnum)=  "5",userdata(tabcontrol)=  "Settings"
+	Button button_sweepFormula_display,pos={52.20,88.20},size={37.80,18.00},proc=button_sweepFormula_display,title="Display"
```

Comment:
@ukos-git I'm not sure why I see this:

![image](https://user-images.githubusercontent.com/5693697/64442930-a50f9380-d085-11e9-9be8-f6f9c6a68a36.png)

Reply:
Did I link the procedure file "MIES_SweepFormula.ipf"  in the "MIES_include" file? That could be it.

Reply:
@ukos-git you are correct; MIES_SweepFormula is missing. The JSON XOP is also required, correct?

Reply:
right. The JSON XOP is parsing the formula into JSON logic. It will presumably also help with our messy WaveNote storage functionality. I am not sure how to integrate it in the install procedure of MIES, though, and the introduced changes will also explicitly require IP8.

Reply:
you will need the XOP and the json_functions.ipf

Reply:
> right. The JSON XOP is parsing the formula into JSON logic. It will presumably also help with our messy WaveNote storage functionality. I am not sure how to integrate it in the install procedure of MIES, though, and the introduced changes will also explicitly require IP8.

@t-b could you help here? I guess we'll need to set up a new submodule for the JSON XOP?

Reply:
@timjarsky Yes, we need to intergrate the JSON-XOP as well. I'll do that together with the IP8-only move.

Reply:
@timjarsky Installer needs an adaptation as well.